### DISCUSSION TITLE
Welcome to speedcord Discussions!

### DISCUSSION THREAD

**Discussion by TAG-Epic** [TAG-Epic/speedcord#32] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by problemcreator** [comment]
Dude how to use GitHub?
And what is the use of it?
And what is speed cord?

**Reply by TAG-Epic** [reply]
Github is a code collaboration/sharing service
You should look on the github help pages if you don't know how to use it
Speedcord is a library to interact with the discord api
